device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-Q64/config.bcm.j2

```diff
@@ -11,16 +12,18 @@
 {%-     if DEVICE_METADATA['localhost']['subtype'] is defined -%}
 {%-         set switch_subtype = DEVICE_METADATA['localhost']['subtype'] -%}
 {%-         if 'dualtor' in switch_subtype.lower() %}
-{%-             set IPinIP_sock = 'sai_tunnel_support=1
-                                   sai_tunnel_underlay_route_mode=1
-                                   host_as_route_disable=1
-                                   l3_ecmp_levels=2' -%}
+{%-             set IPinIP_sock = 'sai_tunnel_support=1\n\r'
```

Comment:
why do we need \n\r here?

Reply:
changed it to \r\n.

the previous code
```
{%-             set IPinIP_sock = 'sai_tunnel_support=1
                                   sai_tunnel_underlay_route_mode=1
                                   host_as_route_disable=1
                                   l3_ecmp_levels=2' -%}
```
will add extra indents after converting that template. 
if not use \r\n, it will be like this
```
{%-             set IPinIP_sock = 'sai_tunnel_support=1
sai_tunnel_underlay_route_mode=1
host_as_route_disable=1
 l3_ecmp_levels=2' -%}
```